### ISSUE TITLE
[cli] Unable to authenticate / STS

### ISSUE THREAD

**Issue by rsmogura** [aws/aws-cdk#11474]
### Reproduction Steps

Any kind of code communicating with AWS: I.e. deploy stack or create context file (i.e. VPC lookup)

### What actually happened?

Deployment of CDK stack ends with credentials errors. 

### Environment

  - **CDK CLI Version  : 1.60**
  - **Framework Version:**
  - **Node.js Version:**§1
  - **OS               :** MacOS
  - **Language (Version):** TypeScript -->

### Other

Verbose shows that issue comes from SDK
``` 
{
  message: 'STS is not activated in this region for account:XYZ. Your account administrator can activate STS in this region using the IAM Console.',
  code: 'RegionDisabledException',
  time: 2020-11-14Z,
  requestId: 'XYZ',
  statusCode: 403,
  retryable: false,
  retryDelay: 40.91864924317652
}
```

**This is regression from 1.59 to 1.60**. The stack can be deployed using 1.59 version of CDK, but not 1.60. I hope this can help to narrow issue. 

This is :bug: Bug Report

**Comment by rsmogura** [comment]
I think I've found problematic commit - https://github.com/aws/aws-cdk/commit/34450b020d0ef7d2a00f1d56bb5e4a31fa8e12af

This change overrides STS endpoints, for me workaround is to specify region in `~/.aws/credentials`

**Comment by rix0rrr** [comment]
I think your issue might have been solved by this commit:

https://github.com/aws/aws-cdk/commit/95c0332395d1203e8b00fda153fe08e70d0387c5

Released in 1.62.0
